README.md

```diff
@@ -10,9 +10,7 @@ For the software developer, these common endpoints and patterns make it easier t
 
 ## Getting started with RNAget
 
-This repository has several files describing the RNAget API specification:
-
-* The [API specification document](https://github.com/ga4gh-rnaseq/schema/blob/master/rnaget.md)
+* The interactive ReDoc [API specification document](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/ga4gh-rnaseq/schema/master/rnaget-openapi.yaml)
 * A [machine-readable](https://github.com/ga4gh-rnaseq/schema/blob/master/rnaget-openapi.yaml) OpenAPI instance of the specification. View in [online visualizer](https://editor.swagger.io/?url=https://raw.githubusercontent.com/ga4gh-rnaseq/schema/master/rnaget-openapi.yaml)
```

Comment:
Maybe we can use relative links for documents within the GitHub repo?

Reply:
I mean, e.g. `A [machine-readable](rnaget-openapi.yaml) OpenAPI instance ...`

Reply:
It might not work, as I'm hosting the human-readable documentation and redoc on Github Pages, so the base URLs are different (https://ga4gh-rnaseq.github.io/schema/ vs. https://github.com/ga4gh-rnaseq/schema)

Reply:
For those it won't work, but I can make the change to relative links for the instances that do work.

Reply:
I see, let's keep them as they are then